api/proto/teleport/legacy/types/types.proto

```diff
@@ -5671,6 +5671,14 @@ message PluginOktaSyncSettings {
   // app. When empty the plugin will fall back to the legacy behaviour of syncing
   // users from the entre organization.
   string app_id = 5;
+
+  // GroupFilters are filters for which Okta groups to synchronize as access lists.
+  // These are globs/regexes.
```

Comment:
Can you put a regex example? Across Teleport, we have several regexp-like languages, and it's never clear if it's a re2 regexp, PCRE, or something custom that expands `*` and doesn't need `.*` for example.

For IaC this godoc will be used to generate the reference documentation.